server/models/customer.js

```diff
@@ -0,0 +1,31 @@
+//customer: jobs: name, description, required skills, 
```

Comment:
GitHub isn't recognizing that you capitalized the file names on your side. To make GitHub realize the change, first change the file name to customers (plural) and then back to Customer - committing the change in between, and the same for the Contractor model.